libraries/AP_HAL_ChibiOS/hwdef/scripts/defaults_bootloader.h

```diff
@@ -35,3 +35,30 @@
 #ifndef AP_WATCHDOG_SAVE_FAULT_ENABLED
 #define AP_WATCHDOG_SAVE_FAULT_ENABLED 0
 #endif
+
+// less LWIP functionality in the bootloader
+#define LWIP_DHCP 0
+#define LWIP_UDP 0
+#define LWIP_PPP 0
+#define LWIP_IGMP 0
+#define LWIP_ALTCP 0
+#define IP_FORWARD 0
+#define LWIP_SINGLE_NETIF 1
+#define SO_REUSE 0
+#define LWIP_SOCKET_POLL 0
+#define LINK_STATS 0
+#define ICMP_STATS 0
+#define IPFRAG_STATS 0
+#define TCP_STATS 0
+#define ARP_PROXYARP_SUPPORT 0
+#define LWIP_HAVE_LOOPIF 0
+#define LWIP_NETIF_LOOPBACK 0
+
+/*
+  we need DMA on H7 to allow for ECC error checking
+  Note that ChibiOS uses #ifdef for STM32_DMA_REQUIRED not #if
+ */
+#if !defined(STM32_DMA_REQUIRED) && defined(STM32H7)
+#define STM32_DMA_REQUIRED 1
```

Comment:
This should be enabled already because the Ethernet MAC uses DMA

Reply:
no, it isn't, as the bootloader includes specifically disable DMA unless we do this